administration/src/cards/extensions/__tests__/StartDayExtension.test.tsx

```diff
@@ -0,0 +1,189 @@
+      const { getByText, getByDisplayValue } = renderWithTranslation(
+        <StartDayExtension.Component {...defaultProps} isValid={false} value={{ startDay }} />,
+        { wrapper }
+      )
+      const datePicker = getByDisplayValue(startDay.format())
+      fireEvent.blur(datePicker)
+      expect(
+        getByText(`Das Startdatum darf nicht weiter als ${minStartDay.format()} in der Vergangenheit liegen.`)
+      ).toBeTruthy()
+    })
+
+    it('should show error message when startDay is too far in the future and touched', () => {
+      const startDay = new PlainDate(2200, 1, 1)
+      const today = PlainDate.fromLocalDate(new Date())
+      const { getByText, getByDisplayValue } = renderWithTranslation(
+        <StartDayExtension.Component {...defaultProps} isValid={false} value={{ startDay }} />,
+        { wrapper }
+      )
+      const datePicker = getByDisplayValue(startDay.format())
+      fireEvent.blur(datePicker)
+      expect(
+        getByText(`Das Startdatum darf nicht weiter als ${today.add(maxCardValidity).format()} in der Zukunft liegen.`)
+      ).toBeTruthy()
+    })
+
+    it('should not show error message when form is invalid but untouched', () => {
+      const { queryByText } = renderWithTranslation(<StartDayExtension.Component {...defaultProps} isValid={false} />, {
+        wrapper,
+      })
+      const errorMessage = queryByText('Bitte geben Sie ein gültiges Startdatum ein, das in der Vergangenheit liegt.')
+      expect(errorMessage).toBeNull()
+    })
+
+    it('should call setValue when date is changed', () => {
+      const { getByDisplayValue } = renderWithTranslation(<StartDayExtension.Component {...defaultProps} />, {
+        wrapper,
+      })
+      const startDayChanged = new PlainDate(2025, 1, 2)
+      const datePicker = getByDisplayValue('01.01.2024')
+      fireEvent.change(datePicker, { target: { value: startDayChanged.format() } })
+      expect(mockSetValue).toHaveBeenCalledWith({ startDay: startDayChanged })
+    })
+
+    it('should set startDay to null if no valid date was typed in', () => {
+      const { getByDisplayValue } = renderWithTranslation(<StartDayExtension.Component {...defaultProps} />, {
+        wrapper,
+      })
+      const datePicker = getByDisplayValue('01.01.2024')
+      fireEvent.change(datePicker, { target: { value: '02' } })
+      expect(mockSetValue).toHaveBeenCalledWith({ startDay: null })
+    })
+
+    describe('getProtobufData', () => {
+      it('should result in correct days since epoch 1970', () => {
+        expect(StartDayExtension.getProtobufData({ startDay: new PlainDate(2022, 1, 1) })).toEqual({
+          extensionStartDay: { startDay: 18993 },
+        })
+      })
+
+      it('should result in minStartDay if no startDay is provided', () => {
+        expect(StartDayExtension.getProtobufData({ startDay: null })).toEqual({
+          extensionStartDay: { startDay: 18424 },
+        })
+      })
+    })
+
+    describe('isValid', () => {
```

Comment:
❓ maybe these tests can be combined with the ones where we check for error messages / no error messages ?
otherwise these tests should be duplicated, because we need to check that the field is invalid, in the same cases where we are checking if an error message is displayed